mne_bids/write.py

```diff
@@ -1317,29 +1365,61 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
+    if trans is not None or landmarks is not None:
         t1_mgh = nib.MGHImage(t1w.dataobj, t1w.affine)
-        vox2ras_tkr = t1_mgh.header.get_vox2ras_tkr()
-        ras2vox_tkr = np.linalg.inv(vox2ras_tkr)
-        mri_landmarks = apply_trans(ras2vox_tkr, mri_landmarks)  # in vox
+
+        if landmarks is not None:
+            if raw is not None and verbose:
+                raise ValueError('Please use either `landmarks` or `raw`, ' +
+                                 'which digitization to use is ambiguous.')
+            if isinstance(landmarks, str):
+                landmarks, coord_frame = read_fiducials(landmarks)
+                landmarks = np.array([landmark['r'] for landmark in
+                                      landmarks], dtype=float)  # unpack
+            else:
+                coords_dict, coord_frame = _get_fid_coords(landmarks.dig)
+                landmarks = np.asarray((coords_dict['lpa'],
+                                        coords_dict['nasion'],
+                                        coords_dict['rpa']))
+            if coord_frame == FIFF.FIFFV_COORD_HEAD:
+                if trans is None:
+                    raise ValueError('Head space landmarks provided, ' +
+                                     '`trans` required')
+                mri_landmarks = _meg_landmarks_to_mri_landmarks(
+                    landmarks, trans)
+                mri_landmarks = _mri_landmarks_to_mri_voxels(
+                    mri_landmarks, t1_mgh)
+            elif coord_frame in (FIFF.FIFFV_MNE_COORD_MRI_VOXEL,
+                                 FIFF.FIFFV_COORD_MRI):
+                if trans is not None:
+                    raise ValueError('`trans` was provided but `landmark` ' +
+                                     'data is in mri space. Please use ' +
+                                     'only one of these.')
+                if coord_frame == FIFF.FIFFV_COORD_MRI:
+                    landmarks = _mri_landmarks_to_mri_voxels(landmarks * 1e3,
+                                                             t1_mgh)
+                mri_landmarks = landmarks
+            else:
+                raise ValueError('Coordinate frame not recognized, ' +
+                                 'found %s' % coord_frame)
+        elif trans is not None:
+            # get trans and ensure it is from head to MRI
+            trans, _ = _get_trans(trans, fro='head', to='mri')
+
+            if not isinstance(raw, BaseRaw):
+                raise ValueError('`raw` must be specified if `trans` ' +
+                                 'is not None')
+
+            # Prepare to write the sidecar JSON
+            # extract MEG landmarks
+            coords_dict, coord_fname = _get_fid_coords(raw.info['dig'])
+            meg_landmarks = np.asarray((coords_dict['lpa'],
+                                        coords_dict['nasion'],
+                                        coords_dict['rpa']))
```

Comment:
why is this lowercase now?

Reply:
It's in the function Alex asked to be used